regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -72,8 +72,11 @@ var CommentReviewView = Backbone.View.extend({
   },
 
   serialize: function() {
+    // TODO(vrajmohan) Add other regs.gov fields
     return {
-      sections: comments.toJSON({docId: this.docId})
+        general_comment:  {
```

Comment:
If I'm reading this correctly, I'm not sure that we want to do this. I assume we're calling this `general_comment` because that's the field expected by the regs.gov api--but we don't actually pass this value through as `general_comment`. We always pass in "See attached comment.pdf". So I actually find it to be a little confusing to use the regs.gov name here, since this isn't the value we're going to send under that field.

This also interacts with validation in a potentially confusing way. If `general_comment` is among the fields that we validate in `sanitize_fields`, it looks like we're always going to pass in a dict of length one, which means we'll pass the `max_length` check--but the length we're checking isn't the length of the `general_comment` string.

Does this make sense? Let me know if I'm misunderstanding.